### DISCUSSION TITLE
Queue feature

### DISCUSSION THREAD

**Discussion by bgalvao** [dailydotdev/daily#478] [Feature Requests]
I love this extension, but let's be honest: it is somewhat disruptive. I am in the middle of work and open a new tab in the browser. Then I see that shiny title that really makes me want to read an article, but! it is not the right time as I am in the middle of a task.

What if dailydev had a queue feature? Kind of like Watch Later playlist on YouTube. Some thoughts on why this would be a good feature, based on personal experience:

- I've been abusing the bookmark feature, but then I end up with a collection of articles I haven't even read or checked whether they're "bookmark-quality".
- the user could relax from FOMO and only open an article when taking a break.
    - it has happened to me that I missed an article from the page upon refreshing dailydev's new tab.
- I don't like to have tabs opened from daily dev waiting for me and cluttering my view and still forgetting about them anyways, so this would be another reason to have a queue feature: to keep articles I want to read in a safe and tidy place.
- you could even collect and display a metric: how many users have queued a given article vs how many have upvoted it. This would capture the aspect of which titles get the attention of readers and which are actually valued by upvotes.

**Comment by idoshamun** [comment]
Oh man, you described it so well!
I think we have it opened already here: https://github.com/dailydotdev/daily/discussions/364
Please let me know if I can close this discussion and move the conversation there. What do you think?

**Reply by bgalvao** [reply]
Yeah, we should close this discussion :). (I'm copying my opening comment over to #364)
